ColorzCore/Raws/Raw.cs

```diff
@@ -15,7 +15,7 @@ class Raw
         public string Name { get; }
         private int length;
         public short Code { get; }
-        public int OffsetMod { get; }

```

Comment:
For compatibility (or, more importantly, the original documentation for EA) reasons, maybe keep this field.

`public int OffsetMod {get;}
public int AlignmentRequirement {get { return OffsetMod; } }
`